collections/requirements.yaml

```diff
@@ -0,0 +1,4 @@
+---
+collections:
+  - community.mysql
+...
```

Comment:
what is this file for?

Reply:
https://aap2-prod-us-east-2.aap.infra.demo.redhat.com/#/jobs/playbook/655973/output

![image](https://github.com/user-attachments/assets/a491dcba-0afe-4bcd-b5a8-b3125988bf5b)

Not sure if it is the correct place to install that collection, but I'm recovering that file as it is working nice like this... if any other aproach is proposed, I can modify without any problem :-)

Reply:
Moved the contents

Reply:
These contents can't be moved, as the collection `community.mysql` must be present when the `main.yml` is importing (and not including) the other playbooks. I'm recovering the `collections/requirements.yml` file to let the AAP download the collection when syncing the project.

Reply:
I'm not using the `collections/requirements.yml` file. I'm changing the `roles` usage by the `include_role` to make it dynamic, as this is what I suppose it's expected.